models/user.js

```diff
@@ -1,17 +1,46 @@
-const mongoose = require('mongoose');
-const {Monitor} = require('./monitor');
+const mongoose = require("mongoose");
+const { Monitor } = require("./monitor");
+const bcrypt = require("bcrypt");
 const Schema = mongoose.Schema;
 
 const User = new Schema({
-    username: { type: String , unique: true },
```

Comment:
I would suggest against using unique in mongoose schema, I have seen this to not work sometime depending on the data processing. I would not take a risk with this.